osu.Game/Localisation/GameplaySettingsStrings.cs

```diff
@@ -104,6 +104,31 @@ public static class GameplaySettingsStrings
         /// </summary>
         public static LocalisableString IncreaseFirstObjectVisibility => new TranslatableString(getKey(@"increase_first_object_visibility"), @"Increase visibility of first object when visual impairment mods are enabled");
 
+        /// <summary>
+        /// "Hide during gameplay"
+        /// </summary>
+        public static LocalisableString HideDuringGameplay => new TranslatableString(getKey(@"hide_during_gameplay"), @"Hide during gameplay");
+
+        /// <summary>
+        /// "Always"
+        /// </summary>
+        public static LocalisableString Always => new TranslatableString(getKey(@"always"), @"Always");
```

Comment:
There's a few duplicate strings here that should be moved into `CommonStrings` imo. Thoughts @peppy ?

Reply:
I intentionally asked them to move these into specific implementations, as they will translate differently in some languages, such as japanese, for certain settings.

They do need proper prefixes in the keys though. This should be "SomethingAlways" not just "Always".

Reply:
I added prefixes to common strings to differenciate between them.